omnipaxos/src/sequence_paxos/mod.rs

```diff
@@ -214,23 +214,21 @@ where
     pub(crate) fn resend_message_timeout(&mut self) {
         match &self.state {
             (Role::Leader, phase) => {
```

Comment:
Do we also need to resend `Prepare` if we are in the prepare phase? I'm thinking in the case where we elect a new leader that sends a prepare to everybody which gets dropped. All servers are still connected so they won't elect a new leader, but we will not make progress because the leader does not complete the prepare phase.

If so then change this code to:
```rust
match &self.state {
     (Role::Leader, Phase::Prepare) => { // resend prepare }
     (Role::Leader, Phase::Accept) => { // your current code }

}
```

Reply:
The code now has the leader resending `Prepare`s  no matter the phase, so long as the `Role` is `Leader`. So this case will be covered. Another question is whether we want the leader to send prepares when their `Phase` is `Recovery` or `None`.

Reply:
Maybe its more clear to explicitly pattern match on the entire state like you suggest?

Reply:
Right, now I remember. Let's keep it this way, because even in the Accept phase, we might need to send `Prepare` to the followers that haven't promised yet.